webrender/Cargo.toml

```diff
@@ -63,10 +63,6 @@ optional = true
 
 [dev-dependencies]
 mozangle = "0.1"
```

Comment:
why is `mozangle` left?

Reply:
Because it is used by the `tests` folder not by the `examples`.